package.json

```diff
@@ -63,7 +64,9 @@
     "semantic-release-slack-bot": "1.6.2",
     "sequelize": "5.21.3",
     "sequelize-fixtures": "1.1.1",
-    "simple-git": "1.65.0"
+    "sequelize-test-helpers": "1.3.2",
+    "simple-git": "1.65.0",
+    "sinon": "9.2.4"
```

Comment:
`sinon` and `chai` are needed for `sequelize-test-helpers` to work properly.
Ideally we should have our own version of `sequelize-test-helpers` with `jest`